internal/node/prereq.go

```diff
@@ -107,28 +106,44 @@ func CheckPrerequisites(config *NodeConfiguration, readonly bool) error {
 
 	// Verify all directories are present
 	for _, r := range *required {
-		sb.WriteString(fmt.Sprintf("Validating - %s [%s]\n", magenta(r.descriptor), cyan(r.directory)))
+		sb.WriteString(fmt.Sprintf("Validating - %s\n", magenta(r.descriptor)))
 
-		allDepsSatified := true
-		for _, f := range r.files {
-			path := func() string {
-				if r.directory == "" {
-					return f.name
-				} else {
-					return filepath.Join(r.directory, f.name)
+		depsFound := 0
+		for _, dir := range r.directories {
+			if dir != "" {
+				sb.WriteString(fmt.Sprintf("\t🔎 Searching - %s \n", cyan(dir)))
```

Comment:
Dumb request, can you make the 🔍 line up with the ✅
![Screenshot 2024-02-05 at 10 41 07 AM](https://github.com/synadia-io/nex/assets/15827604/af4ce546-38c5-4662-aee1-5b6cd6c09c28)